### PR TITLE
ref(node): Clarify usage of onFatalError option in node

### PR DESCRIPTION
**Pull request by kamilogorek**
Fixes https://github.com/getsentry/sentry-javascript/issues/1661#issuecomment-718200921
Fixes https://github.com/getsentry/sentry-javascript/issues/2796#issuecomment-718204061

### CODE REVIEW

File: src/platforms/node/common/configuration/integrations/default-integrations.mdx [hunk 1]
```diff
@@ -69,6 +69,13 @@ Available options:
 _Import name: `Sentry.Integrations.OnUncaughtException`_
 
 This integration attaches a global uncaught exception handler. It can be modified to provide a custom shutdown function.
+The `onFatalError` option is meant to perform a cleanup before the process exits, not fully prevent it.
```
Comment (lobsterkatie) [inline src/platforms/node/common/configuration/integrations/default-integrations.mdx:72]:
```suggestion
The `onFatalError` option is meant to perform a cleanup before the process exits, not fully prevent it from exiting.
```

File: src/platforms/node/common/configuration/integrations/default-integrations.mdx [hunk 2]
```diff
@@ -69,6 +69,13 @@ Available options:
...
+
+<Alert level="warning" title="Note">
+
+Be aware that if you are overwriting this setting, you will lose the default implementation, which handles draining queued events before exiting. If you want to see how it works and recreates in your custom code, see the implementation of [`logAndExitProcess`](https://github.com/getsentry/sentry-javascript/blob/master/packages/node/src/handlers.ts) function.
```
Comment (lobsterkatie) [inline src/platforms/node/common/configuration/integrations/default-integrations.mdx:76]:
```suggestion
Be aware that if you overwrite this setting, you will lose the default implementation, which handles draining queued events before exiting. If you want to see how it works in order to be able to recreate it in your code, see the implementation of the [`logAndExitProcess`](https://github.com/getsentry/sentry-javascript/blob/master/packages/node/src/handlers.ts) function.
```

Comment (MimiDumpling) [inline src/platforms/node/common/configuration/integrations/default-integrations.mdx:76]:
```suggestion
Be aware that if you are overwriting this setting, you will lose the default implementation, which handles draining queued events before exiting. If you want to examine how it works and then recreate it in your code, see the implementation of the [`logAndExitProcess`](https://github.com/getsentry/sentry-javascript/blob/master/packages/node/src/handlers.ts) function.
```

**Review by MimiDumpling — APPROVED** [review]
Thanks, Kamil! 🌻

File: src/platforms/node/common/configuration/integrations/default-integrations.mdx [hunk 3]
```diff
@@ -69,6 +69,13 @@ Available options:
 _Import name: `Sentry.Integrations.OnUncaughtException`_
 
 This integration attaches a global uncaught exception handler. It can be modified to provide a custom shutdown function.
+The `onFatalError` option is meant to perform a cleanup before the process exits, not fully prevent it from exiting.
+
+<Alert level="warning" title="Note">
+
+Be aware that if you overwrite this setting, you will lose the default implementation, which handles draining queued events before exiting. If you want to see how it works in order to be able to recreate it in your code, see the implementation of the [`logAndExitProcess`](https://github.com/getsentry/sentry-javascript/blob/master/packages/node/src/handlers.ts) function.
```
Comment (kamilogorek) [inline src/platforms/node/common/configuration/integrations/default-integrations.mdx:76]:
```suggestion
Be aware that if you overwrite this setting, you will lose the default implementation, which handles draining queued events before exiting. If you want to examine how it works in order to be able to recreate it in your code, see the implementation of the [`logAndExitProcess`](https://github.com/getsentry/sentry-javascript/blob/master/packages/node/src/handlers.ts) function.
```

**Comment by kamilogorek** [comment]
Applied both suggestions. Thanks! :)

**Pull request merged**